specification/synapse/resource-manager/readme.md

```diff
@@ -28,17 +28,40 @@ These are the global settings for the Azure Synapse Analytics API.
 description: Azure Synapse Analytics Management Client
 openapi-type: arm
 azure-arm: true
-tag: package-composite-v2
+tag: package-2023-05
```

Comment:
I'm curious why not all the APIs are added stable version, like `sparkConfiguration` and `kustoPool`. This tag is used for generating SDK. It will cause SDK breaking change without these APIs.

Reply:
sparkConfiguration and kustoPool are not in the stable version as expected. these are not in the previous stable version 2021-06-01 either. 
This tag is used for generating SDK - are you referring to changing default tag?
I tried keeping the default tag as same and modifying swaggers in package-composite-v2, but it gives 
Swagger Avocado failed - The default tag does not contains the latest API version. Please make sure the latest api version swaggers are in the default tag.

Please suggest what needs to be done. We have been waiting for getting this merged. cc @vishnumathi